.eslintrc

```diff
@@ -22,14 +22,7 @@
         "endOfLine": "auto"
       }
     ],
-    "max-len": [
-      "error",
-      {
-        "code": 120,
-        "ignoreUrls": true,
-        "ignorePattern": "*className=([\\s\\S]*?)*" // Ignore classnames
-      }
-    ]
```

Comment:
I have disabled this rule since we have many places in the code where the length of lines exceeds 120 characters (even 240 didn't work here).

Reply:
Nope, we shouldn't remove the rule because in UI, it is already ignoring the `className` patterns. For rest of the code, we can have this rule enabled for them.